src/commands/commerce/extension/map.ts

```diff
@@ -96,17 +96,10 @@ export class MapExtension extends SfdxCommand {
             fResult = forceDataSoql(`SELECT Id FROM WebStore WHERE Name='${storeName}'`, userName);
             if (fResult !== undefined && fResult.result !== undefined) {
                 if (fResult.result.totalSize > 1) {
-                    throw new SfdxError(
-                        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-                        msgs.getMessage('extension.map.multiple', [storeName])
-                    );
+                    throw new SfdxError(msgs.getMessage('extension.map.multiple', [storeName]));
```

Comment:
nit: i found it confusing on first glance what message 'extension.map.multiple' might contain. Would we consider renaming it to something more descriptive like: nit: extension.map.errMultipleStoreWithSameName